review/fileToReview.js

```diff
@@ -0,0 +1,73 @@
+
+var domain = "bank.local.fr"
+
+/**
+ * @description Fetch transactions recursively
+ * @param {string} fromDate The maximum date of transactions to return
+ * @param {string} authorization Authorization header to authent the user
+ * @param {jws} jws Jws token, mandatory if we get one from login request
```

Comment:
`jwsToken` would probably be more clear
The type is wrong here, is it a string?
The description is confusing, could you me more precise?
```suggestion
 * @param {string} jwsToken Jws token, mandatory if we get one from login request
```